### ISSUE TITLE
Replace nbshell with shell_plus

### ISSUE THREAD

**Issue by smk4664** [nautobot/nautobot#951]
### Environment
* Nautobot version: 1.2.x

### Proposed Functionality
Since Nautobot 1.2 introduces django_extensions I would like to see nbshell replaced with shell_plus.

### Use Case
Shell_plus auto-imports models like nbshell, but also supports importing models for plugins. There are other neat features such as iPython support.

### Database Changes
None

### External Dependencies

None.

### TODO

- Remove `nbshell` management command
- Update docs to inform to use `shell` instead of `nbshell`
- Remove `nbshell` invoke command
- Add `shell` management command (in Core) which would start `shell_plus`, overloading Django's built-in shell command
- Add `shell` invoke command

**Comment by jathanism** [comment]
This can be accomplished by replacing the contents of `nautobot/extras/management/commands/nbshell.py` with:

```python
from django_extensions.management.commands.shell_plus import Command
```

**Comment by glennmatthews** [comment]
Implemented for 2.0 via #3181.
